docs/Documentation/Configuration/Version-Changes/Migration-2-3.md

```diff
@@ -850,3 +851,29 @@ items:
 ```
 
 </div>
+
+### 3.0.0-DEV-444 - Simple Item QuestItem :thunder_cloud_rain:
+
+The way to identify "QuestItems" which can be stored in the backpack has been changed.
+
+For the `simple` Item that means the `quest-item` tag is used instead of the lore.
+When you still want to display that information you still need to add the lore line.
```

Comment:
But we have still the language key for quest Items, we should also get rid of this